eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -45,15 +45,15 @@ library
     build-depends:
         base >=4.14 && <4.19,
         bytestring,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.6,
```

Change not needed, it will work fine with older version
```suggestion
        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
```